data/docs/instrumentation/nextjs.mdx

```diff
@@ -0,0 +1,315 @@
+// this enables the API to record telemetry
+sdk.start()
+
+// gracefully shut down the SDK on process exit
+process.on('SIGTERM', () => {
+  sdk.shutdown()
+    .then(() => console.log('Tracing terminated'))
+    .catch((error) => console.log('Error terminating tracing', error))
+    .finally(() => process.exit(0));
+});
+```
+```
+
+**Step 4.** Once we are done with the above configurations, you can run the collector service with the following command:
+```bash
+./otelcol-contrib --config ./config.yaml
+```
+
+**Step 5.** Start running your application and wait for few seconds to start receiving instrumentation data on the SigNoz Cloud. 
+
+</TabItem>
+</Tabs>
+
+## Send traces to SigNoz Self-Host
+
+If you're trying to send instrumentation data to SigNoz self-hosted way, the only minor thing that you'd be required is to change the `exporterOptions` in the `tracing.js` file
+
+```jsx
+// tracing.js
+'use strict'
+const process = require('process');
+const opentelemetry = require('@opentelemetry/sdk-node');
+const { getNodeAutoInstrumentations } = require('@opentelemetry/auto-instrumentations-node');
+const { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-http');
+const { Resource } = require('@opentelemetry/resources');
+const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+
+// // Add otel logging when debugging
+// const { diag, DiagConsoleLogger, DiagLogLevel } = require('@opentelemetry/api');
+// diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG);
+
+const exporterOptions = {
+  url: 'https://ingest.[data-region].signoz.cloud:443/v1/traces', // use your own data region 
+}
+
+const traceExporter = new OTLPTraceExporter(exporterOptions);
+const sdk = new opentelemetry.NodeSDK({
+  traceExporter,
+  instrumentations: [getNodeAutoInstrumentations()],
+  resource: new Resource({
+    [SemanticResourceAttributes.SERVICE_NAME]: 'SigNoz-Nextjs-Example'
+  })
+});
+
+// initialize the SDK and register with the OpenTelemetry API
+// this enables the API to record telemetry
+sdk.start()
+
+// gracefully shut down the SDK on process exit
+process.on('SIGTERM', () => {
+  sdk.shutdown()
+    .then(() => console.log('Tracing terminated'))
+    .catch((error) => console.log('Error terminating tracing', error))
+    .finally(() => process.exit(0));
+});
+```
+
+Again, `http://localhost:4318/v1/traces` is the default url for sending your tracing data. We are assuming you have installed SigNoz on your localhost. Based on your environment, you can update it accordingly. It should be in the following format:
+
+`http://<IP of SigNoz backend>:4318/v1/traces`
+
+**NOTE:** Remember to allow incoming requests to port 4318 of machine where SigNoz backend is hosted.
+
+Once you're done with this, add your required changes in the `otel-config.yml` file which can be generally found [here](https://github.com/SigNoz/signoz/blob/develop/deploy/docker/clickhouse-setup/otel-collector-config.yaml).
+
```

Comment:
Add Validating traces section here just before conclusion section:

Checkout this comment: https://github.com/SigNoz/signoz-web/pull/305#discussion_r1651984968

Reply:
Should we try to add this before/after the conclusion section?